<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Investigating taxa of interest | Environmental diversity of microbial eukaryotes</title>
  <meta name="description" content="This is the metabarcoding lab for the Uppsala University course: Diversity and Evolution of Microbial Eukaryotes." />
  <meta name="generator" content="bookdown 0.24.3 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Investigating taxa of interest | Environmental diversity of microbial eukaryotes" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://uu-microbial-eukaryotes.github.io/Metabarcoding_lab/" />
  
  <meta property="og:description" content="This is the metabarcoding lab for the Uppsala University course: Diversity and Evolution of Microbial Eukaryotes." />
  <meta name="github-repo" content="UU-Microbial-Eukaryotes/Metabarcoding_lab" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Investigating taxa of interest | Environmental diversity of microbial eukaryotes" />
  
  <meta name="twitter:description" content="This is the metabarcoding lab for the Uppsala University course: Diversity and Evolution of Microbial Eukaryotes." />
  

<meta name="author" content="Lab designed by Mahwash Jamy" />


<meta name="date" content="2021-11-18" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="exploring-microbial-diversity.html"/>
<link rel="next" href="patterns-of-microbial-community-composition.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/d3-3.5.5/d3.min.js"></script>
<link href="libs/d3tree2-0.1/d3tree2.css" rel="stylesheet" />
<script src="libs/d3tree2-binding-0.1/d3tree2.js"></script>

<script>
/* ========================================================================
 * Bootstrap: transition.js v3.3.7
 * http://getbootstrap.com/javascript/#transitions
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */


+function ($) {
  'use strict';

  // CSS TRANSITION SUPPORT (Shoutout: http://www.modernizr.com/)
  // ============================================================

  function transitionEnd() {
    var el = document.createElement('bootstrap')

    var transEndEventNames = {
      WebkitTransition : 'webkitTransitionEnd',
      MozTransition    : 'transitionend',
      OTransition      : 'oTransitionEnd otransitionend',
      transition       : 'transitionend'
    }

    for (var name in transEndEventNames) {
      if (el.style[name] !== undefined) {
        return { end: transEndEventNames[name] }
      }
    }

    return false // explicit for ie8 (  ._.)
  }

  // http://blog.alexmaccaw.com/css-transitions
  $.fn.emulateTransitionEnd = function (duration) {
    var called = false
    var $el = this
    $(this).one('bsTransitionEnd', function () { called = true })
    var callback = function () { if (!called) $($el).trigger($.support.transition.end) }
    setTimeout(callback, duration)
    return this
  }

  $(function () {
    $.support.transition = transitionEnd()

    if (!$.support.transition) return

    $.event.special.bsTransitionEnd = {
      bindType: $.support.transition.end,
      delegateType: $.support.transition.end,
      handle: function (e) {
        if ($(e.target).is(this)) return e.handleObj.handler.apply(this, arguments)
      }
    }
  })

}(jQuery);
</script>
<script>
/* ========================================================================
 * Bootstrap: collapse.js v3.3.7
 * http://getbootstrap.com/javascript/#collapse
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */

/* jshint latedef: false */

+function ($) {
  'use strict';

  // COLLAPSE PUBLIC CLASS DEFINITION
  // ================================

  var Collapse = function (element, options) {
    this.$element      = $(element)
    this.options       = $.extend({}, Collapse.DEFAULTS, options)
    this.$trigger      = $('[data-toggle="collapse"][href="#' + element.id + '"],' +
                           '[data-toggle="collapse"][data-target="#' + element.id + '"]')
    this.transitioning = null

    if (this.options.parent) {
      this.$parent = this.getParent()
    } else {
      this.addAriaAndCollapsedClass(this.$element, this.$trigger)
    }

    if (this.options.toggle) this.toggle()
  }

  Collapse.VERSION  = '3.3.7'

  Collapse.TRANSITION_DURATION = 350

  Collapse.DEFAULTS = {
    toggle: true
  }

  Collapse.prototype.dimension = function () {
    var hasWidth = this.$element.hasClass('width')
    return hasWidth ? 'width' : 'height'
  }

  Collapse.prototype.show = function () {
    if (this.transitioning || this.$element.hasClass('in')) return

    var activesData
    var actives = this.$parent && this.$parent.children('.panel').children('.in, .collapsing')

    if (actives && actives.length) {
      activesData = actives.data('bs.collapse')
      if (activesData && activesData.transitioning) return
    }

    var startEvent = $.Event('show.bs.collapse')
    this.$element.trigger(startEvent)
    if (startEvent.isDefaultPrevented()) return

    if (actives && actives.length) {
      Plugin.call(actives, 'hide')
      activesData || actives.data('bs.collapse', null)
    }

    var dimension = this.dimension()

    this.$element
      .removeClass('collapse')
      .addClass('collapsing')[dimension](0)
      .attr('aria-expanded', true)

    this.$trigger
      .removeClass('collapsed')
      .attr('aria-expanded', true)

    this.transitioning = 1

    var complete = function () {
      this.$element
        .removeClass('collapsing')
        .addClass('collapse in')[dimension]('')
      this.transitioning = 0
      this.$element
        .trigger('shown.bs.collapse')
    }

    if (!$.support.transition) return complete.call(this)

    var scrollSize = $.camelCase(['scroll', dimension].join('-'))

    this.$element
      .one('bsTransitionEnd', $.proxy(complete, this))
      .emulateTransitionEnd(Collapse.TRANSITION_DURATION)[dimension](this.$element[0][scrollSize])
  }

  Collapse.prototype.hide = function () {
    if (this.transitioning || !this.$element.hasClass('in')) return

    var startEvent = $.Event('hide.bs.collapse')
    this.$element.trigger(startEvent)
    if (startEvent.isDefaultPrevented()) return

    var dimension = this.dimension()

    this.$element[dimension](this.$element[dimension]())[0].offsetHeight

    this.$element
      .addClass('collapsing')
      .removeClass('collapse in')
      .attr('aria-expanded', false)

    this.$trigger
      .addClass('collapsed')
      .attr('aria-expanded', false)

    this.transitioning = 1

    var complete = function () {
      this.transitioning = 0
      this.$element
        .removeClass('collapsing')
        .addClass('collapse')
        .trigger('hidden.bs.collapse')
    }

    if (!$.support.transition) return complete.call(this)

    this.$element
      [dimension](0)
      .one('bsTransitionEnd', $.proxy(complete, this))
      .emulateTransitionEnd(Collapse.TRANSITION_DURATION)
  }

  Collapse.prototype.toggle = function () {
    this[this.$element.hasClass('in') ? 'hide' : 'show']()
  }

  Collapse.prototype.getParent = function () {
    return $(this.options.parent)
      .find('[data-toggle="collapse"][data-parent="' + this.options.parent + '"]')
      .each($.proxy(function (i, element) {
        var $element = $(element)
        this.addAriaAndCollapsedClass(getTargetFromTrigger($element), $element)
      }, this))
      .end()
  }

  Collapse.prototype.addAriaAndCollapsedClass = function ($element, $trigger) {
    var isOpen = $element.hasClass('in')

    $element.attr('aria-expanded', isOpen)
    $trigger
      .toggleClass('collapsed', !isOpen)
      .attr('aria-expanded', isOpen)
  }

  function getTargetFromTrigger($trigger) {
    var href
    var target = $trigger.attr('data-target')
      || (href = $trigger.attr('href')) && href.replace(/.*(?=#[^\s]+$)/, '') // strip for ie7

    return $(target)
  }


  // COLLAPSE PLUGIN DEFINITION
  // ==========================

  function Plugin(option) {
    return this.each(function () {
      var $this   = $(this)
      var data    = $this.data('bs.collapse')
      var options = $.extend({}, Collapse.DEFAULTS, $this.data(), typeof option == 'object' && option)

      if (!data && options.toggle && /show|hide/.test(option)) options.toggle = false
      if (!data) $this.data('bs.collapse', (data = new Collapse(this, options)))
      if (typeof option == 'string') data[option]()
    })
  }

  var old = $.fn.collapse

  $.fn.collapse             = Plugin
  $.fn.collapse.Constructor = Collapse


  // COLLAPSE NO CONFLICT
  // ====================

  $.fn.collapse.noConflict = function () {
    $.fn.collapse = old
    return this
  }


  // COLLAPSE DATA-API
  // =================

  $(document).on('click.bs.collapse.data-api', '[data-toggle="collapse"]', function (e) {
    var $this   = $(this)

    if (!$this.attr('data-target')) e.preventDefault()

    var $target = getTargetFromTrigger($this)
    var data    = $target.data('bs.collapse')
    var option  = data ? 'toggle' : $this.data()

    Plugin.call($target, option)
  })

}(jQuery);
</script>
<script>
window.initializeCodeFolding = function(show) {

  // handlers for show-all and hide all
  $("#rmd-show-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('show');
    });
  });
  $("#rmd-hide-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('hide');
    });
  });

  // index for unique code element ids
  var currentIndex = 1;

  // select all R code blocks
  var rCodeBlocks = $('pre.sourceCode, pre.r, pre.python, pre.bash, pre.sql, pre.cpp, pre.stan');
  rCodeBlocks.each(function() {

    // create a collapsable div to wrap the code in
    var div = $('<div class="collapse r-code-collapse"></div>');
    if (show)
      div.addClass('in');
    var id = 'rcode-643E0F36' + currentIndex++;
    div.attr('id', id);
    $(this).before(div);
    $(this).detach().appendTo(div);

    // add a show code button right above
    var showCodeText = $('<span>' + (show ? 'Hide' : 'Code') + '</span>');
    var showCodeButton = $('<button type="button" class="btn btn-default btn-xs code-folding-btn pull-right"></button>');
    showCodeButton.append(showCodeText);
    showCodeButton
        .attr('data-toggle', 'collapse')
        .attr('data-target', '#' + id)
        .attr('aria-expanded', show)
        .attr('aria-controls', id);

    var buttonRow = $('<div class="row"></div>');
    var buttonCol = $('<div class="col-md-12"></div>');

    buttonCol.append(showCodeButton);
    buttonRow.append(buttonCol);

    div.before(buttonRow);

    // update state of button on show/hide
    div.on('hidden.bs.collapse', function () {
      showCodeText.text('Code');
    });
    div.on('show.bs.collapse', function () {
      showCodeText.text('Hide');
    });
  });

}

</script>
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
.row { display: flex; }
.collapse { display: none; }
.in { display:block }
</style>
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Lab on Metabarcoding</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Before you start the lab</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#learning-outcomes"><i class="fa fa-check"></i><b>1.1</b> Learning outcomes</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#motivation"><i class="fa fa-check"></i><b>1.2</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-data.html"><a href="the-data.html"><i class="fa fa-check"></i><b>2</b> The data</a><ul>
<li class="chapter" data-level="2.1" data-path="the-data.html"><a href="the-data.html#getting-a-feel-for-your-data"><i class="fa fa-check"></i><b>2.1</b> Getting a feel for your data</a></li>
<li class="chapter" data-level="2.2" data-path="the-data.html"><a href="the-data.html#what-do-these-otus-represent-anyway"><i class="fa fa-check"></i><b>2.2</b> What do these OTUs represent anyway?</a><ul>
<li class="chapter" data-level="2.2.1" data-path="the-data.html"><a href="the-data.html#scenario-1"><i class="fa fa-check"></i><b>2.2.1</b> Scenario 1</a></li>
<li class="chapter" data-level="2.2.2" data-path="the-data.html"><a href="the-data.html#scenario-2"><i class="fa fa-check"></i><b>2.2.2</b> Scenario 2</a></li>
<li class="chapter" data-level="2.2.3" data-path="the-data.html"><a href="the-data.html#scenario-3"><i class="fa fa-check"></i><b>2.2.3</b> Scenario 3</a></li>
<li class="chapter" data-level="2.2.4" data-path="the-data.html"><a href="the-data.html#summary-so-far"><i class="fa fa-check"></i><b>2.2.4</b> Summary so far</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html"><i class="fa fa-check"></i><b>3</b> Exploring microbial diversity</a><ul>
<li class="chapter" data-level="3.1" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#identifying-otus"><i class="fa fa-check"></i><b>3.1</b> Identifying OTUs</a></li>
<li class="chapter" data-level="3.2" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#marine-microbial-diversity"><i class="fa fa-check"></i><b>3.2</b> Marine microbial diversity</a><ul>
<li class="chapter" data-level="3.2.1" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#dominant-supergroups-in-the-chosen-stations"><i class="fa fa-check"></i><b>3.2.1</b> Dominant supergroups in the chosen stations</a></li>
<li class="chapter" data-level="3.2.2" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#dominant-major-lineages-in-global-marine-waters"><i class="fa fa-check"></i><b>3.2.2</b> Dominant major lineages in global marine waters</a></li>
<li class="chapter" data-level="3.2.3" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#novel-diversity"><i class="fa fa-check"></i><b>3.2.3</b> Novel diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="investigating-taxa-of-interest.html"><a href="investigating-taxa-of-interest.html"><i class="fa fa-check"></i><b>4</b> Investigating taxa of interest</a><ul>
<li class="chapter" data-level="4.1" data-path="investigating-taxa-of-interest.html"><a href="investigating-taxa-of-interest.html#symbiodinium"><i class="fa fa-check"></i><b>4.1</b> <em>Symbiodinium</em></a></li>
<li class="chapter" data-level="4.2" data-path="investigating-taxa-of-interest.html"><a href="investigating-taxa-of-interest.html#chytrids"><i class="fa fa-check"></i><b>4.2</b> Chytrids</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html"><i class="fa fa-check"></i><b>5</b> Patterns of microbial community composition</a><ul>
<li class="chapter" data-level="5.1" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html#oceans"><i class="fa fa-check"></i><b>5.1</b> Oceans</a></li>
<li class="chapter" data-level="5.2" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html#comparing-oceans-lakes-and-soils"><i class="fa fa-check"></i><b>5.2</b> Comparing oceans, lakes, and soils</a></li>
<li class="chapter" data-level="5.3" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html#fin"><i class="fa fa-check"></i><b>5.3</b> Fin</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/UU-Microbial-Eukaryotes/Metabarcoding_lab" target="blank">GitHub repository</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Environmental diversity of microbial eukaryotes</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="investigating-taxa-of-interest" class="section level1">
<h1><span class="header-section-number">4</span> Investigating taxa of interest</h1>
<hr />
<p>Metabarcoding data can also be used to investigate the biodiversity and ecology of taxa of interest.</p>
<div id="symbiodinium" class="section level2">
<h2><span class="header-section-number">4.1</span> <em>Symbiodinium</em></h2>
<p>In this part of the lab, we will focus on one particular genus, <em>Symbiodinium</em>, and investigate its distribution across the six stations (coral and open ocean). <em>Symbiodinium</em> is a genus of photosynthetic dinoflagellates. These micro-algae, colloquially called “zooxanthellae” are most well-known for being symbionts and the primary producers in reef-building corals. They are thus ecologically and economically important. There has been much research carried out on Symbiodinium biology and ecology in recent yeras in order to understand global coral reef decline.</p>
<p><strong>Questions:</strong><br />
<strong>22. Do you expect to find <em>Symbiodinium</em> in one or both environment types? Explain your reasoning.</strong></p>
<p>Consider the following OTUs from station 49 (coral reef) which have been classified as <em>Symbiodinium</em>.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb19-1"><a href="investigating-taxa-of-interest.html#cb19-1"></a><span class="co"># station 49</span></span>
<span id="cb19-2"><a href="investigating-taxa-of-interest.html#cb19-2"></a><span class="fu">cat</span> data/vsearch_49.tsv <span class="kw">|</span> <span class="fu">grep</span> <span class="st">&#39;Symbiodinium&#39;</span> <span class="kw">|</span> <span class="fu">tr</span> <span class="st">&#39;|&#39;</span> <span class="st">&#39;\t&#39;</span> <span class="kw">|</span> <span class="fu">cut</span> -f 1,2,10,11</span></code></pre></div>
<pre><code>## otu_3007;366 100.0   SymbiodiniumCladeA  sp.
## otu_3567;1   98.4    SymbiodiniumCladeD  sp.
## otu_432448;1 97.7    SymbiodiniumCladeA  sp.
## otu_5241;692 100.0   SymbiodiniumCladeC  sp.
## otu_162983;2 100.0   SymbiodiniumCladeC  sp.
## otu_13108;6  80.8    SymbiodiniumCladeD  sp.
## otu_539869;1 89.9    SymbiodiniumCladeE  californium
## otu_288404;6 96.9    SymbiodiniumCladeD  sp.
## otu_107980;1 98.4    SymbiodiniumCladeA  sp.</code></pre>
<p>Column 1 represents the OTU, Column 2 represents % similarity to closest reference, Columns 3 and 4 represent genus and species respectively.</p>
<p><strong>Questions:</strong><br />
<strong>23. How confident can you be that otu_3007;366 is a <em>Symbiodinium</em>? Why?</strong><br />
<strong>24. How confident can you be that otu_539869;1 is a <em>Symbiodinium</em>? Why?</strong></p>
<p>To resolve this, we will use the Evolutionary Placement Algorithm to “place” sequences in a reference tree. This is a very useful technique for classifying millions of short environmental reads or OTUs. Figure 7 shows an overview of the process.</p>
<p><br></p>
<p><img src="images/epa.png" alt="Fig 7: Phylogenetic placement (source:Bik et al. 2012)" />
<br></p>
<ol style="list-style-type: decimal">
<li>For this, we need to first generate an 18S phylogeny from reference sequences from PR2. This has already been done for us. The tree below shows <em>Symbiodinium</em> sequences rooted with other dinoflagellates.</li>
</ol>
<p><img src="images/ref.png" alt="ref: tree" />
<br></p>
<ol start="2" style="list-style-type: decimal">
<li>We then “place” the OTUs from station 49 above on the reference tree using EPA. This algorithm decides the best location for each OTU without rebuilding the tree from scratch again. This has also been done for us, and the OTUs are shown in red colour.</li>
</ol>
<div class="figure">
<img src="images/ref-49.png" alt="" />
<p class="caption">station 49, coral</p>
</div>
<p><strong>Questions:</strong><br />
<strong>25. How many OTUs are actually Symbiodinium? (These are OTUs that fall in the <em>Symbiodinium</em> clade.)</strong><br />
<strong>26. Are the results consistent with your answers to questions 23 and 24?</strong></p>
<p>Similar analyses have been carried out for you for the other stations. View the trees here below.</p>
<div class="figure">
<img src="images/ref-46.png" alt="" />
<p class="caption">station 46, coral</p>
</div>
<div class="figure">
<img src="images/ref-121.png" alt="" />
<p class="caption">station 121, coral</p>
</div>
<div class="figure">
<img src="images/ref-45.png" alt="" />
<p class="caption">station 45, open ocean</p>
</div>
<div class="figure">
<img src="images/ref-50.png" alt="" />
<p class="caption">station 50, open ocean</p>
</div>
<div class="figure">
<img src="images/ref-126.png" alt="" />
<p class="caption">station 126, open ocean</p>
</div>
<p><strong>Questions:</strong><br />
<strong>26. Are the results consistent with your answer to question 22?</strong></p>
<p><br></p>
<p>Thus far, we have looked at presence/absence of <em>Symbiodinium</em> in coral reef waters and open ocean waters. We can also investigate if there is any difference in abundance (i.e. number of <em>Symbiodinium</em> reads from each of the stations).</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="investigating-taxa-of-interest.html#cb21-1"></a><span class="co"># read data</span></span>
<span id="cb21-2"><a href="investigating-taxa-of-interest.html#cb21-2"></a>sym_abundance &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;data/sym_abundance.txt&quot;</span>, <span class="dt">header =</span> T, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>)</span>
<span id="cb21-3"><a href="investigating-taxa-of-interest.html#cb21-3"></a></span>
<span id="cb21-4"><a href="investigating-taxa-of-interest.html#cb21-4"></a><span class="co"># order stations in the order we wangt to appear in the graph</span></span>
<span id="cb21-5"><a href="investigating-taxa-of-interest.html#cb21-5"></a>sym_abundance<span class="op">$</span>stations &lt;-<span class="st"> </span><span class="kw">factor</span>(sym_abundance<span class="op">$</span>stations, <span class="kw">c</span>(<span class="st">&quot;station_45&quot;</span>, <span class="st">&quot;station_46&quot;</span>, <span class="st">&quot;station_50&quot;</span>, <span class="st">&quot;station_49&quot;</span>, <span class="st">&quot;station_121&quot;</span>, <span class="st">&quot;station_126&quot;</span>))</span>
<span id="cb21-6"><a href="investigating-taxa-of-interest.html#cb21-6"></a></span>
<span id="cb21-7"><a href="investigating-taxa-of-interest.html#cb21-7"></a><span class="co"># custom theme</span></span>
<span id="cb21-8"><a href="investigating-taxa-of-interest.html#cb21-8"></a><span class="kw">require</span>(ggplot2)</span>
<span id="cb21-9"><a href="investigating-taxa-of-interest.html#cb21-9"></a></span>
<span id="cb21-10"><a href="investigating-taxa-of-interest.html#cb21-10"></a>cust_theme &lt;-<span class="st"> </span><span class="kw">theme_dark</span>() <span class="op">+</span><span class="st"> </span></span>
<span id="cb21-11"><a href="investigating-taxa-of-interest.html#cb21-11"></a><span class="st">  </span><span class="kw">theme</span>(<span class="dt">axis.text.y =</span> <span class="kw">element_text</span>(<span class="dt">angle=</span><span class="dv">90</span>, <span class="dt">hjust=</span><span class="fl">0.5</span>), <span class="dt">axis.text.x =</span> <span class="kw">element_text</span>(<span class="dt">angle=</span><span class="dv">90</span>, <span class="dt">vjust=</span><span class="fl">0.5</span>, <span class="dt">hjust=</span><span class="dv">1</span>), <span class="dt">strip.text.x =</span> <span class="kw">element_text</span>(<span class="dt">angle=</span><span class="dv">180</span>),</span>
<span id="cb21-12"><a href="investigating-taxa-of-interest.html#cb21-12"></a>        <span class="dt">panel.grid.major.x =</span> <span class="kw">element_blank</span>(), <span class="dt">panel.grid.minor.x =</span> <span class="kw">element_blank</span>(), </span>
<span id="cb21-13"><a href="investigating-taxa-of-interest.html#cb21-13"></a>        <span class="dt">strip.placement =</span> <span class="st">&quot;outside&quot;</span>, </span>
<span id="cb21-14"><a href="investigating-taxa-of-interest.html#cb21-14"></a>        <span class="dt">panel.background =</span> <span class="kw">element_rect</span>(<span class="dt">fill =</span> <span class="st">&quot;white&quot;</span>),</span>
<span id="cb21-15"><a href="investigating-taxa-of-interest.html#cb21-15"></a>        <span class="dt">strip.background =</span><span class="kw">element_blank</span>(), </span>
<span id="cb21-16"><a href="investigating-taxa-of-interest.html#cb21-16"></a>        <span class="dt">panel.spacing =</span> <span class="kw">unit</span>(<span class="dv">0</span>, <span class="st">&quot;lines&quot;</span>),</span>
<span id="cb21-17"><a href="investigating-taxa-of-interest.html#cb21-17"></a>        <span class="dt">axis.ticks =</span> <span class="kw">element_blank</span>())</span>
<span id="cb21-18"><a href="investigating-taxa-of-interest.html#cb21-18"></a></span>
<span id="cb21-19"><a href="investigating-taxa-of-interest.html#cb21-19"></a><span class="co"># plot</span></span>
<span id="cb21-20"><a href="investigating-taxa-of-interest.html#cb21-20"></a><span class="kw">ggplot</span>(sym_abundance, <span class="kw">aes</span>(<span class="dt">x =</span> stations, <span class="dt">y =</span> rel_abundance)) <span class="op">+</span><span class="st"> </span></span>
<span id="cb21-21"><a href="investigating-taxa-of-interest.html#cb21-21"></a><span class="st">    </span><span class="kw">geom_bar</span>(<span class="kw">aes</span>(<span class="dt">fill =</span> env_type), <span class="dt">color=</span><span class="st">&quot;white&quot;</span>, <span class="dt">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="dt">position =</span> <span class="st">&quot;dodge&quot;</span>) <span class="op">+</span></span>
<span id="cb21-22"><a href="investigating-taxa-of-interest.html#cb21-22"></a><span class="st">    </span><span class="kw">xlab</span>(<span class="st">&quot;&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">ylab</span>(<span class="st">&quot;Symbiodinium abundance (% of reads)&quot;</span>) <span class="op">+</span></span>
<span id="cb21-23"><a href="investigating-taxa-of-interest.html#cb21-23"></a><span class="st">    </span><span class="kw">ggtitle</span>(<span class="st">&quot;Symbiodinium abundance&quot;</span>) <span class="op">+</span></span>
<span id="cb21-24"><a href="investigating-taxa-of-interest.html#cb21-24"></a><span class="st">    </span>cust_theme</span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
<p><strong>Questions:</strong><br />
<strong>27. Is there a trend in abundance of <em>Symbiodinium</em>?</strong><br />
<strong>28. Based on these results, what can we say about the lifestyle of <em>Symbiodinium</em>? Does it exclusively live through symbiosis with corals?</strong><br />
<strong>29. Does <em>Symdiodinium</em> have a free-living stage, or does it form symbioses with other organisms? We can test this hypothesis by looking at the presense and abundance of <em>Symbiodinium</em> in different size fractions on an online server. (Follow the instructions below and answer this question. Recall that <em>Symbiodinium</em> cells are usually between 6-13 microns in diameter.).</strong></p>
<p>The <a href="https://oba.mio.osupytheas.fr/ocean-atlas/">Ocean Barcode Atlas</a> is an online web service for rapidly analysing the biogeography of taxa using the entire Tara Oceans dataset. Go to the website and click on <em>Community ecological analysis</em>. In the <em>taxonomy</em> field, type in “Symbiodiniaceae”, untick computation of beta diversity, set maps and bubble plots to 1, and submit your job.</p>
<p>Select the following size fractions: 0.8-5 microns, 5-20 microns, 20-180 microns, 180-2000 microns. What does this tell us about the life cycle of <em>Symbiodinium</em>?</p>
</div>
<div id="chytrids" class="section level2">
<h2><span class="header-section-number">4.2</span> Chytrids</h2>
<p>Chytridiomycota (informally chytrids) are fungi with flagellated zoospores which have recently been uncovered as important parasites or saprotrophs of diatoms in marine systems (Figure 8).</p>
<div class="figure">
<img src="images/chytrid.png" alt="" />
<p class="caption">Fig 8: Scanning electron microscographs of pennate diatoms infected by chytrid-like fungi which are coloured in red (image from Kilias et al. 2020. <em>Communications Biology</em>)</p>
</div>
<p><strong>Question:</strong><br />
<strong>30. Using the Ocean Barcode Atlas again, can you determine which temperature range do Chytridiomycota prefer? Are they more abundant in tropical waters?</strong></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="exploring-microbial-diversity.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="patterns-of-microbial-community-composition.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UU-Microbial-Eukaryotes/Metabarcoding_lab/edit/develop/03-taxa_of_interest.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
