<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Patterns of microbial community composition | Environmental diversity of microbial eukaryotes</title>
  <meta name="description" content="This is the metabarcoding lab for the Uppsala University course: Diversity and Evolution of Microbial Eukaryotes." />
  <meta name="generator" content="bookdown 0.24.3 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Patterns of microbial community composition | Environmental diversity of microbial eukaryotes" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://uu-microbial-eukaryotes.github.io/Metabarcoding_lab/" />
  
  <meta property="og:description" content="This is the metabarcoding lab for the Uppsala University course: Diversity and Evolution of Microbial Eukaryotes." />
  <meta name="github-repo" content="UU-Microbial-Eukaryotes/Metabarcoding_lab" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Patterns of microbial community composition | Environmental diversity of microbial eukaryotes" />
  
  <meta name="twitter:description" content="This is the metabarcoding lab for the Uppsala University course: Diversity and Evolution of Microbial Eukaryotes." />
  

<meta name="author" content="Lab designed by Mahwash Jamy" />


<meta name="date" content="2021-11-18" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="investigating-taxa-of-interest.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/d3-3.5.5/d3.min.js"></script>
<link href="libs/d3tree2-0.1/d3tree2.css" rel="stylesheet" />
<script src="libs/d3tree2-binding-0.1/d3tree2.js"></script>

<script>
/* ========================================================================
 * Bootstrap: transition.js v3.3.7
 * http://getbootstrap.com/javascript/#transitions
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */


+function ($) {
  'use strict';

  // CSS TRANSITION SUPPORT (Shoutout: http://www.modernizr.com/)
  // ============================================================

  function transitionEnd() {
    var el = document.createElement('bootstrap')

    var transEndEventNames = {
      WebkitTransition : 'webkitTransitionEnd',
      MozTransition    : 'transitionend',
      OTransition      : 'oTransitionEnd otransitionend',
      transition       : 'transitionend'
    }

    for (var name in transEndEventNames) {
      if (el.style[name] !== undefined) {
        return { end: transEndEventNames[name] }
      }
    }

    return false // explicit for ie8 (  ._.)
  }

  // http://blog.alexmaccaw.com/css-transitions
  $.fn.emulateTransitionEnd = function (duration) {
    var called = false
    var $el = this
    $(this).one('bsTransitionEnd', function () { called = true })
    var callback = function () { if (!called) $($el).trigger($.support.transition.end) }
    setTimeout(callback, duration)
    return this
  }

  $(function () {
    $.support.transition = transitionEnd()

    if (!$.support.transition) return

    $.event.special.bsTransitionEnd = {
      bindType: $.support.transition.end,
      delegateType: $.support.transition.end,
      handle: function (e) {
        if ($(e.target).is(this)) return e.handleObj.handler.apply(this, arguments)
      }
    }
  })

}(jQuery);
</script>
<script>
/* ========================================================================
 * Bootstrap: collapse.js v3.3.7
 * http://getbootstrap.com/javascript/#collapse
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */

/* jshint latedef: false */

+function ($) {
  'use strict';

  // COLLAPSE PUBLIC CLASS DEFINITION
  // ================================

  var Collapse = function (element, options) {
    this.$element      = $(element)
    this.options       = $.extend({}, Collapse.DEFAULTS, options)
    this.$trigger      = $('[data-toggle="collapse"][href="#' + element.id + '"],' +
                           '[data-toggle="collapse"][data-target="#' + element.id + '"]')
    this.transitioning = null

    if (this.options.parent) {
      this.$parent = this.getParent()
    } else {
      this.addAriaAndCollapsedClass(this.$element, this.$trigger)
    }

    if (this.options.toggle) this.toggle()
  }

  Collapse.VERSION  = '3.3.7'

  Collapse.TRANSITION_DURATION = 350

  Collapse.DEFAULTS = {
    toggle: true
  }

  Collapse.prototype.dimension = function () {
    var hasWidth = this.$element.hasClass('width')
    return hasWidth ? 'width' : 'height'
  }

  Collapse.prototype.show = function () {
    if (this.transitioning || this.$element.hasClass('in')) return

    var activesData
    var actives = this.$parent && this.$parent.children('.panel').children('.in, .collapsing')

    if (actives && actives.length) {
      activesData = actives.data('bs.collapse')
      if (activesData && activesData.transitioning) return
    }

    var startEvent = $.Event('show.bs.collapse')
    this.$element.trigger(startEvent)
    if (startEvent.isDefaultPrevented()) return

    if (actives && actives.length) {
      Plugin.call(actives, 'hide')
      activesData || actives.data('bs.collapse', null)
    }

    var dimension = this.dimension()

    this.$element
      .removeClass('collapse')
      .addClass('collapsing')[dimension](0)
      .attr('aria-expanded', true)

    this.$trigger
      .removeClass('collapsed')
      .attr('aria-expanded', true)

    this.transitioning = 1

    var complete = function () {
      this.$element
        .removeClass('collapsing')
        .addClass('collapse in')[dimension]('')
      this.transitioning = 0
      this.$element
        .trigger('shown.bs.collapse')
    }

    if (!$.support.transition) return complete.call(this)

    var scrollSize = $.camelCase(['scroll', dimension].join('-'))

    this.$element
      .one('bsTransitionEnd', $.proxy(complete, this))
      .emulateTransitionEnd(Collapse.TRANSITION_DURATION)[dimension](this.$element[0][scrollSize])
  }

  Collapse.prototype.hide = function () {
    if (this.transitioning || !this.$element.hasClass('in')) return

    var startEvent = $.Event('hide.bs.collapse')
    this.$element.trigger(startEvent)
    if (startEvent.isDefaultPrevented()) return

    var dimension = this.dimension()

    this.$element[dimension](this.$element[dimension]())[0].offsetHeight

    this.$element
      .addClass('collapsing')
      .removeClass('collapse in')
      .attr('aria-expanded', false)

    this.$trigger
      .addClass('collapsed')
      .attr('aria-expanded', false)

    this.transitioning = 1

    var complete = function () {
      this.transitioning = 0
      this.$element
        .removeClass('collapsing')
        .addClass('collapse')
        .trigger('hidden.bs.collapse')
    }

    if (!$.support.transition) return complete.call(this)

    this.$element
      [dimension](0)
      .one('bsTransitionEnd', $.proxy(complete, this))
      .emulateTransitionEnd(Collapse.TRANSITION_DURATION)
  }

  Collapse.prototype.toggle = function () {
    this[this.$element.hasClass('in') ? 'hide' : 'show']()
  }

  Collapse.prototype.getParent = function () {
    return $(this.options.parent)
      .find('[data-toggle="collapse"][data-parent="' + this.options.parent + '"]')
      .each($.proxy(function (i, element) {
        var $element = $(element)
        this.addAriaAndCollapsedClass(getTargetFromTrigger($element), $element)
      }, this))
      .end()
  }

  Collapse.prototype.addAriaAndCollapsedClass = function ($element, $trigger) {
    var isOpen = $element.hasClass('in')

    $element.attr('aria-expanded', isOpen)
    $trigger
      .toggleClass('collapsed', !isOpen)
      .attr('aria-expanded', isOpen)
  }

  function getTargetFromTrigger($trigger) {
    var href
    var target = $trigger.attr('data-target')
      || (href = $trigger.attr('href')) && href.replace(/.*(?=#[^\s]+$)/, '') // strip for ie7

    return $(target)
  }


  // COLLAPSE PLUGIN DEFINITION
  // ==========================

  function Plugin(option) {
    return this.each(function () {
      var $this   = $(this)
      var data    = $this.data('bs.collapse')
      var options = $.extend({}, Collapse.DEFAULTS, $this.data(), typeof option == 'object' && option)

      if (!data && options.toggle && /show|hide/.test(option)) options.toggle = false
      if (!data) $this.data('bs.collapse', (data = new Collapse(this, options)))
      if (typeof option == 'string') data[option]()
    })
  }

  var old = $.fn.collapse

  $.fn.collapse             = Plugin
  $.fn.collapse.Constructor = Collapse


  // COLLAPSE NO CONFLICT
  // ====================

  $.fn.collapse.noConflict = function () {
    $.fn.collapse = old
    return this
  }


  // COLLAPSE DATA-API
  // =================

  $(document).on('click.bs.collapse.data-api', '[data-toggle="collapse"]', function (e) {
    var $this   = $(this)

    if (!$this.attr('data-target')) e.preventDefault()

    var $target = getTargetFromTrigger($this)
    var data    = $target.data('bs.collapse')
    var option  = data ? 'toggle' : $this.data()

    Plugin.call($target, option)
  })

}(jQuery);
</script>
<script>
window.initializeCodeFolding = function(show) {

  // handlers for show-all and hide all
  $("#rmd-show-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('show');
    });
  });
  $("#rmd-hide-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('hide');
    });
  });

  // index for unique code element ids
  var currentIndex = 1;

  // select all R code blocks
  var rCodeBlocks = $('pre.sourceCode, pre.r, pre.python, pre.bash, pre.sql, pre.cpp, pre.stan');
  rCodeBlocks.each(function() {

    // create a collapsable div to wrap the code in
    var div = $('<div class="collapse r-code-collapse"></div>');
    if (show)
      div.addClass('in');
    var id = 'rcode-643E0F36' + currentIndex++;
    div.attr('id', id);
    $(this).before(div);
    $(this).detach().appendTo(div);

    // add a show code button right above
    var showCodeText = $('<span>' + (show ? 'Hide' : 'Code') + '</span>');
    var showCodeButton = $('<button type="button" class="btn btn-default btn-xs code-folding-btn pull-right"></button>');
    showCodeButton.append(showCodeText);
    showCodeButton
        .attr('data-toggle', 'collapse')
        .attr('data-target', '#' + id)
        .attr('aria-expanded', show)
        .attr('aria-controls', id);

    var buttonRow = $('<div class="row"></div>');
    var buttonCol = $('<div class="col-md-12"></div>');

    buttonCol.append(showCodeButton);
    buttonRow.append(buttonCol);

    div.before(buttonRow);

    // update state of button on show/hide
    div.on('hidden.bs.collapse', function () {
      showCodeText.text('Code');
    });
    div.on('show.bs.collapse', function () {
      showCodeText.text('Hide');
    });
  });

}

</script>
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
.row { display: flex; }
.collapse { display: none; }
.in { display:block }
</style>
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="toc-logo"><a href="./"><img src="images/logo.png" width="150"></a><li>
<li><a href="./">Lab on Metabarcoding</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Before you start the lab</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#learning-outcomes"><i class="fa fa-check"></i><b>1.1</b> Learning outcomes</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#motivation"><i class="fa fa-check"></i><b>1.2</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-data.html"><a href="the-data.html"><i class="fa fa-check"></i><b>2</b> The data</a><ul>
<li class="chapter" data-level="2.1" data-path="the-data.html"><a href="the-data.html#getting-a-feel-for-your-data"><i class="fa fa-check"></i><b>2.1</b> Getting a feel for your data</a></li>
<li class="chapter" data-level="2.2" data-path="the-data.html"><a href="the-data.html#what-do-these-otus-represent-anyway"><i class="fa fa-check"></i><b>2.2</b> What do these OTUs represent anyway?</a><ul>
<li class="chapter" data-level="2.2.1" data-path="the-data.html"><a href="the-data.html#scenario-1"><i class="fa fa-check"></i><b>2.2.1</b> Scenario 1</a></li>
<li class="chapter" data-level="2.2.2" data-path="the-data.html"><a href="the-data.html#scenario-2"><i class="fa fa-check"></i><b>2.2.2</b> Scenario 2</a></li>
<li class="chapter" data-level="2.2.3" data-path="the-data.html"><a href="the-data.html#scenario-3"><i class="fa fa-check"></i><b>2.2.3</b> Scenario 3</a></li>
<li class="chapter" data-level="2.2.4" data-path="the-data.html"><a href="the-data.html#summary-so-far"><i class="fa fa-check"></i><b>2.2.4</b> Summary so far</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html"><i class="fa fa-check"></i><b>3</b> Exploring microbial diversity</a><ul>
<li class="chapter" data-level="3.1" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#identifying-otus"><i class="fa fa-check"></i><b>3.1</b> Identifying OTUs</a></li>
<li class="chapter" data-level="3.2" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#marine-microbial-diversity"><i class="fa fa-check"></i><b>3.2</b> Marine microbial diversity</a><ul>
<li class="chapter" data-level="3.2.1" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#dominant-supergroups-in-the-chosen-stations"><i class="fa fa-check"></i><b>3.2.1</b> Dominant supergroups in the chosen stations</a></li>
<li class="chapter" data-level="3.2.2" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#dominant-major-lineages-in-global-marine-waters"><i class="fa fa-check"></i><b>3.2.2</b> Dominant major lineages in global marine waters</a></li>
<li class="chapter" data-level="3.2.3" data-path="exploring-microbial-diversity.html"><a href="exploring-microbial-diversity.html#novel-diversity"><i class="fa fa-check"></i><b>3.2.3</b> Novel diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="investigating-taxa-of-interest.html"><a href="investigating-taxa-of-interest.html"><i class="fa fa-check"></i><b>4</b> Investigating taxa of interest</a><ul>
<li class="chapter" data-level="4.1" data-path="investigating-taxa-of-interest.html"><a href="investigating-taxa-of-interest.html#symbiodinium"><i class="fa fa-check"></i><b>4.1</b> <em>Symbiodinium</em></a></li>
<li class="chapter" data-level="4.2" data-path="investigating-taxa-of-interest.html"><a href="investigating-taxa-of-interest.html#chytrids"><i class="fa fa-check"></i><b>4.2</b> Chytrids</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html"><i class="fa fa-check"></i><b>5</b> Patterns of microbial community composition</a><ul>
<li class="chapter" data-level="5.1" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html#oceans"><i class="fa fa-check"></i><b>5.1</b> Oceans</a></li>
<li class="chapter" data-level="5.2" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html#comparing-oceans-lakes-and-soils"><i class="fa fa-check"></i><b>5.2</b> Comparing oceans, lakes, and soils</a></li>
<li class="chapter" data-level="5.3" data-path="patterns-of-microbial-community-composition.html"><a href="patterns-of-microbial-community-composition.html#fin"><i class="fa fa-check"></i><b>5.3</b> Fin</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/UU-Microbial-Eukaryotes/Metabarcoding_lab" target="blank">GitHub repository</a></li>
<li><a href="https://creativecommons.org/licenses/by/4.0/" target="blank">© CC-BY 2021</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Environmental diversity of microbial eukaryotes</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="patterns-of-microbial-community-composition" class="section level1">
<h1><span class="header-section-number">5</span> Patterns of microbial community composition</h1>
<hr />
<div id="oceans" class="section level2">
<h2><span class="header-section-number">5.1</span> Oceans</h2>
<p>What factors explain the microbial community compositions in different environmental samples? We will now return to our six Tara Ocean stations (coral and non-coral), and quantify the differences in microbial communities between the stations, and cluster them based on their differences. The measure of dissimilarity we will use is Bray-Curtis, which is commonly used in ecology. You do not need to know how it works, however if you are interested in a short, easy to read summary, see <a href="https://www.statology.org/bray-curtis-dissimilarity/">here</a></p>
<p>Importantly, the measure is between 0 and 1. Identical communities would have a Bray-Curtis dissimilarity of 0, while communities which share no species would have a Bray-Curtis dissimilarity of 1.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="patterns-of-microbial-community-composition.html#cb22-1"></a><span class="co"># load required package</span></span>
<span id="cb22-2"><a href="patterns-of-microbial-community-composition.html#cb22-2"></a><span class="kw">library</span>(vegan)</span></code></pre></div>
<pre><code>## Loading required package: permute</code></pre>
<pre><code>## Loading required package: lattice</code></pre>
<pre><code>## This is vegan 2.5-6</code></pre>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="patterns-of-microbial-community-composition.html#cb26-1"></a><span class="co"># read data</span></span>
<span id="cb26-2"><a href="patterns-of-microbial-community-composition.html#cb26-2"></a>matrix &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;data/chosen_stations_filtered_otus.tsv&quot;</span>, <span class="dt">header=</span>T, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> <span class="dv">1</span>)</span>
<span id="cb26-3"><a href="patterns-of-microbial-community-composition.html#cb26-3"></a></span>
<span id="cb26-4"><a href="patterns-of-microbial-community-composition.html#cb26-4"></a><span class="co"># view the data we are working with. The table contains read counts for each OTU in every station. Each row = OTU, each column = station.</span></span>
<span id="cb26-5"><a href="patterns-of-microbial-community-composition.html#cb26-5"></a><span class="kw">head</span>(matrix)</span></code></pre></div>
<pre><code>##       station46 station49 station121 station45 station50 station126
## otu_1    160143    191068      63806    165918     48066      29592
## otu_2       505       651        322      1079       446        690
## otu_3     14895     10798      84622     37216     63754      51660
## otu_4    172454     20426      38846    188886     31332     467641
## otu_5     36236     12180     126136     70310     37317     213247
## otu_6    114638     64892      12028    187475    116723       7397</code></pre>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="patterns-of-microbial-community-composition.html#cb28-1"></a><span class="co"># transpose data so that rows and columns are switched</span></span>
<span id="cb28-2"><a href="patterns-of-microbial-community-composition.html#cb28-2"></a>matrix_t &lt;-<span class="st"> </span><span class="kw">t</span>(matrix)</span>
<span id="cb28-3"><a href="patterns-of-microbial-community-composition.html#cb28-3"></a></span>
<span id="cb28-4"><a href="patterns-of-microbial-community-composition.html#cb28-4"></a><span class="co"># normalise data</span></span>
<span id="cb28-5"><a href="patterns-of-microbial-community-composition.html#cb28-5"></a>matrix_t_norm &lt;-<span class="st"> </span><span class="kw">decostand</span>(matrix_t, <span class="dt">method =</span> <span class="st">&quot;total&quot;</span>) </span>
<span id="cb28-6"><a href="patterns-of-microbial-community-composition.html#cb28-6"></a></span>
<span id="cb28-7"><a href="patterns-of-microbial-community-composition.html#cb28-7"></a><span class="co"># calculate Bray-Curtis distance and cluster the stations</span></span>
<span id="cb28-8"><a href="patterns-of-microbial-community-composition.html#cb28-8"></a>matrix.bc.dist &lt;-<span class="st"> </span><span class="kw">vegdist</span>(matrix_t_norm, <span class="dt">method =</span> <span class="st">&quot;bray&quot;</span>)</span>
<span id="cb28-9"><a href="patterns-of-microbial-community-composition.html#cb28-9"></a>matrix.bc.clust &lt;-<span class="st"> </span><span class="kw">hclust</span>(matrix.bc.dist, <span class="dt">method =</span> <span class="st">&quot;average&quot;</span>)</span>
<span id="cb28-10"><a href="patterns-of-microbial-community-composition.html#cb28-10"></a><span class="kw">plot</span>(matrix.bc.clust, <span class="dt">ylab =</span> <span class="st">&quot;Bray-Curtis dissimilarity&quot;</span>)</span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-11-1.png" width="672" /></p>
<p><strong>Questions:</strong><br />
<strong>31. What explains the clustering that we see?</strong></p>
</div>
<div id="comparing-oceans-lakes-and-soils" class="section level2">
<h2><span class="header-section-number">5.2</span> Comparing oceans, lakes, and soils</h2>
<p>Finally, we are going to zoom out and compare the broad microbial communities from ocean, lake, and soil samples. To get an overview of the communities, we will phylogenetically place the metabarcoding sequences onto a reference phylogeny as we did for <em>Symbiodinium</em> previously. This time however, we will place OTUs on a global eukaryotic phylogeny with 155 annotated, reference 18S sequences (these sequences are derived from the PR2 18S database) (Figure 8). Note that because this tree is based on 18S sequences only, it is not entirely consistent with what we know from phylogenomics. However, this does not impact phylogenetic placement.</p>
<div class="figure">
<img src="images/ref.tre.png" alt="" />
<p class="caption">Fig 8: Reference 18S phylogeny of eukaryotes with major groups annotated.</p>
</div>
<p><br></p>
<p>The three datasets we will be comparing today are the Malaspina dataset (a global marine expedition like the Tara Oceans), a freshwater dataset from Lake Baikal, and a dataset from Neotropical forest soils. We’re opting for the Malapspina dataset instead of the Tara because like the soil and freshwater data, it covers the longer, V4 region instead of the V9 region.</p>
<p>Let’s have a cursory look at how many OTUs are in each dataset (NB: normally, we would normalize data first before comparisons, but for the purpose of this lab, it is okay to proceed with un-normalised data.)</p>
<p><strong>Marine - Malaspina</strong></p>
<div class="sourceCode" id="cb29"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb29-1"><a href="patterns-of-microbial-community-composition.html#cb29-1"></a><span class="co"># marine</span></span>
<span id="cb29-2"><a href="patterns-of-microbial-community-composition.html#cb29-2"></a><span class="fu">cat</span> data/malaspina.fasta <span class="kw">|</span> <span class="fu">grep</span> -c <span class="st">&#39;&gt;&#39;</span></span></code></pre></div>
<pre><code>## 8217</code></pre>
<p><strong>Neotropical forest soils</strong></p>
<div class="sourceCode" id="cb31"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb31-1"><a href="patterns-of-microbial-community-composition.html#cb31-1"></a><span class="co"># soils</span></span>
<span id="cb31-2"><a href="patterns-of-microbial-community-composition.html#cb31-2"></a><span class="fu">cat</span> data/neotropical-soils.fasta <span class="kw">|</span> <span class="fu">grep</span> -c <span class="st">&#39;&gt;&#39;</span></span></code></pre></div>
<pre><code>## 28474</code></pre>
<p><strong>Freshwater - lake Baikal</strong></p>
<div class="sourceCode" id="cb33"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb33-1"><a href="patterns-of-microbial-community-composition.html#cb33-1"></a><span class="co"># freshwater</span></span>
<span id="cb33-2"><a href="patterns-of-microbial-community-composition.html#cb33-2"></a><span class="fu">cat</span> data/baikal.fasta <span class="kw">|</span> <span class="fu">grep</span> -c <span class="st">&#39;&gt;&#39;</span></span></code></pre></div>
<pre><code>## 1125</code></pre>
<p>The phylogenetic placement has already been carried out. Download the placement files from <a href="https://github.com/UU-Microbial-Eukaryotes/Metabarcoding_lab/">GitHub</a>. To do so, first download the GitHub repository as a zip file: i.e. <em>Code</em> (in top-right corner in green) –&gt; <em>Download ZIP</em>. After saving, unzip the folder and navigate to <em>Metabarcoding_lab-master</em> –&gt; <em>placement_files</em>. You will see three files with the extension <em>.jplace</em>.</p>
<p>Now for the fun part! We will use the interactive <a href="https://itol.embl.de/upload.cgi">Tree of Life website</a> to visualize the difference between these three environments. Simply load each <em>.jplace</em> file (using the button “Choose file”, selecting the file, and click “Upload”). I recommend using three tabs so you can view all files at the same time.</p>
<p>You will now see a phylogenetic tree and a control panel. Select the following options in the control panel:</p>
<ul>
<li><em>Mode options</em> –&gt; <em>Branch lengths</em> –&gt; <em>Ignore</em> (this will make it easier to view the placements)<br />
</li>
<li><em>Label options</em> –&gt; <em>Font style</em> –&gt; <em>26 px</em> (or higher so you can view the tip labels)<br />
</li>
<li><em>Datasets</em> –&gt; <em>Phylogenetic placements</em> –&gt; <em>On</em></li>
</ul>
<p>You will now see bubbles on the tree branches. Each bubble represents OTU placements on a particular branch. The bigger the bubble, the more OTUs belong to that particular lineage!</p>
<p><strong>Questions:</strong><br />
<strong>32. What are the most dominant lineages in soils and freshwater samples?</strong><br />
<strong>33. In which environment are fungi the most dominant?</strong><br />
<strong>34. Which lineages are dominant in the oceans, but absent in freshwater and soils?</strong><br />
<strong>35. Are there lineages present in freshwater/soils but absent in oceans? Which ones?</strong>
<strong>36. List any other observations you find interesting!</strong></p>
</div>
<div id="fin" class="section level2">
<h2><span class="header-section-number">5.3</span> Fin</h2>
<p>Discuss your answers with your TA :)</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="investigating-taxa-of-interest.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UU-Microbial-Eukaryotes/Metabarcoding_lab/edit/develop/04-patterns.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
